brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
Again, if you have _strong_  preference for number of frames, I could control it from BluePy side and pass number of frames here.